ui/perfherder/compare/CompareTable.jsx

```diff
@@ -120,11 +118,17 @@ export default class CompareTable extends React.PureComponent {
             </th>
             <th className="table-width-lg">Base</th>
             {/* empty for less than/greater than data */}
-            <th className="table-width-sm" />
+            <th
+              className="table-width-sm"
+              aria-label="Comparison table row header"
+            />
             <th className="table-width-lg">New</th>
             <th className="table-width-lg">Delta</th>
             {/* empty for progress bars (magnitude of difference) */}
-            <th className="table-width-lg" />
+            <th
+              className="table-width-lg"
+              aria-label="Comparison table row header"
```

Comment:
This aria-label text isn't helpful because it's not describing what the column should contain. This one should say something like 'magnitude of difference'.